File: src/test/resources/expectedfolder/admin/LoginRequest.cs
using System;
using com.google.devtools.java2csharp.testclasses;
using com.google.devtools.java2csharp.testclasses.admin;

namespace com.google.devtools.java2csharp.testclasses.admin
{
	public class LoginRequest : Message
	{
		private string username;

		private string password;

		protected internal LoginRequest()
			: base()
		{
		}

		public LoginRequest(Header header, string username, string password)
			: base(header)
		{
			this.username = username;
			this.password = password;
		}

		public virtual string Username
		{
			get
			{
				return username;
			}
			set
			{
				string username = value;
				this.username = username;
			}
		}

		public virtual string Password
		{
			get
			{
				return password;
			}
			set
			{
				string password = value;
				this.password = password;
			}
		}
	}
}

File: src/test/java/com/google/devtools/java2csharp/testclasses/admin/LoginRequest.java
package com.google.devtools.java2csharp.testclasses.admin;

import com.google.devtools.java2csharp.testclasses.Header;

public class LoginRequest extends Message {
	private String username;

	private String password;

	protected LoginRequest() {
		super();
	}

	public LoginRequest(Header header, String username,
			String password) {
		super(header);
		this.username = username;
		this.password = password;
	}

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}
}